src/lib/components/index/acm-paths.svelte

```diff
@@ -3,87 +3,89 @@
 </script>
 
 <section>
-  <div class="paths-intro">
-    <h2 class="headers size-lg">Find the path you want to take</h2>
-    <p class="size-xs">
-      Our programs were designed to allow students to explore the various fields within tech and
-      discover what aligns with their interests the most. By joining any of them, students will be
-      given access to exclusive workshops and projects that will build, enhance, and strengthen
-      their technical and professional skills.
-    </p>
-  </div>
+  <div class="container">
+    <div class="paths-intro">
+      <h2 class="brand-header size-lg">Discover your own path</h2>
+
+      <p class="size-sm">
+        Paths were designed to allow students to explore the various fields within tech, and
+        discover what aligns with their interests the most. Our paths offer interactive workshops
+        that enhance technical knowledge, as well as fun projects to strengthen their abilities.
+      </p>
+    </div>
+
+    <div class="paths-list">
+      {#each pinnedPaths as { title, slug, picture, color } (slug)}
+        <a class="path-item" target="_self" href={`/paths#${slug}`}>
+          <img src={picture} alt={`${slug} badge`} />
 
-  <div class="paths-list">
-    {#each pinnedPaths as { title, slug, picture, color } (slug)}
-      <a class="path-item" target="_self" href={`/paths#${slug}`}>
-        <img src={picture} alt={`${slug}-logo`} />
-        <p class="size-md headers">
-          acm<span class="brand-em" style:--brand-color={color}>
+          <p class="size-md brand-bold" style:--brand-color={color}>
```

Comment:
The span from line 21 in the old build is now incorporated into the p tag since we don't need to add ACM in the front anymore